modules/installation-creating-aws-security.adoc

```diff
@@ -6,9 +6,9 @@
 [id="installation-creating-aws-security_{context}"]
 = Creating security group and roles in AWS
 
-You must create security groups and roles in Amazon Web Services (AWS) for your
-{product-title} cluster to use. The easiest way to create these components is
-to modify the provided CloudFormation template.
+You must create security groups and roles in Amazon Web Services (AWS) for your {product-title} cluster to use.
+
+You can use the provided CloudFormation template and a custom parameter file, to create a stack of AWS resources. The stack represents the security groups and roles required for your {product-title} cluster.
```

Comment:
See previous comments for similar content about "file, to"

Reply:
Resolved. I have also now applied active voice to the second sentence.